common/git-hooks/post-commit

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Called by "git commit" with no arguments. The hook cannot alter
+# the success of the commit that invoked it.
+
+# Verify that all recently committed files have the correct copyright header
+# if not, then the correct header will be added or replace the old
+
+unstagedChanges=$(git diff)
+
+# stash all unstaged changes, including untracked files
+if [ "$unstagedChanges" != "" ]; then
+git stash -u
+fi
+
+if $(node ./common/scripts/copyright-linter.js) ; then
+  if [ "$(git diff --name-only HEAD)" != "" ]; then
+  # the linter succeeded and changes were made, add them to the previous commit
+  # may be empty if all previous changes were to the linter headers
+  git commit -a --amend --allow-empty
```

Comment:
Silence this commit to prevent confusing output of there being two commits with the same message when in actuality it is rolled into one commit.